### ISSUE TITLE
How to set the architecture of the Flutter SDK?

### ISSUE THREAD

**Issue by nilsreichardt** [fluttertools/fvm#416]
<img width="936" alt="image" src="https://user-images.githubusercontent.com/24459435/167962292-bd9db1dc-fd07-406c-b842-7601663e2989.png">

The Flutter SDK is now available for x86 and arm64. How can I specify which architecture I want to use when using the `fvm install VERSION` command?

**Comment by imSaharukh** [comment]
I'm also looking for this

**Comment by leoafarias** [comment]
@nilsreichardt good question… to be honest fvm just clones the GitHub repo and dependencies get installed and built on demand… so it should just work

let me know if you are having any issues, or if what I am saying is not the case

**Comment by imSaharukh** [comment]
> @nilsreichardt good question… to be honest fvm just clones the GitHub repo and dependencies get installed and built on demand… so it should just work
> 
> 
> 
> let me know if you are having any issues, or if what I am saying is not the case 

so if i want to install the arm64 version of flutter 3.0.0 then what should be the command?

**Comment by leoafarias** [comment]
`fvm use 3.0.0` to attach to a project and run

**Comment by imSaharukh** [comment]
> `fvm use 3.0.0` to attach to a project and run or just `fvm use stable` will work also since 3.0.0 is the latest stable version

But the issue is 3.0.0 is available in two different architecture

**Comment by leoafarias** [comment]
@imSaharukh only clones the repo… flutter tools handles all the installation of dependencies and building itself.

fvm also does distinguish between windows or Linux from an install perspective… which is why it should not be a problem.. 

But again I’d you actually have an issue let me know… and I can take a look at it

**Comment by jeroen-meijer** [comment]
I did some research and I think this is a non-issue. 😄

I just set up a brand new M1 Pro MacBook, and before I even installed Rosetta, I downloaded Dart through `brew` (with the right architecture), installed `fvm` and used `fvm global 3.0.0` to install Flutter 3. After that, `fvm flutter doctor` ran perfectly, without having Rosetta installed.

This leads me to believe everything runs natively on Apple Silicon 👍🏻
